doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,577 @@
-////
-This is the base template for Hazelcast integration module guides.
+:github-address: https://github.com/hazelcast-guides/ec2-cluster
+:templates-url: templates:ROOT:page$/
+:aws-plugin-url: https://github.com/hazelcast/hazelcast-aws
+:tab-header-console: AWS Console
+:tab-header-cli: AWS CLI
 
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
+= Deploy Hazelcast Cluster on AWS EC2
 
-:github-address: [github-repo-address-of-your-guide]
+This guide will get you started to create a Hazelcast cluster on EC2.
 
-:templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+== What You’ll Learn
 
-// Use this relative url if you are going to publish the guide on the guides site.
-// Note that this url will not work locally and raise asciidoctor errors.
-// So, complete the guide with the above url and set the below one just before 
-// publishing on the guides site.
-//
-// :templates-url: templates:ROOT:page$/
+In this guide, you will create two EC2 instances with Hazelcast members and see them connecting each other and
+forming a cluster using {aws-plugin-url}[Hazelcast AWS plugin].
 
-= Title
+[NOTE]
+====
+Steps 1 to 4 cover the creation and the configuration of VPC, internet gateway, route table, subnet and IAM role.
+If you already have these configured you can skip them. However, make sure that you have these properties before
+creating instances:
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+* [x] Have an IAM role with `ec2:DescribeInstances` permission
+* [x] Have a security group with at least port 5701 open
+* [x] Able to establish SSH connections to instances
+====
 
-include::{templates-url}/link-to-repo.adoc[]
+== 1) Create and Configure a VPC
```

Comment:
Using `.` instead of `)` is more stylish. WDYT? 

1. Create and Configure a VPC
1.1. Create a VPC